checks/apps/gromacs/gromacs_large.py

```diff
@@ -0,0 +1,161 @@
+import os
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+@rfm.simple_test
+class lumi_gromacs_large(rfm.RunOnlyRegressionTest):
+    '''The test uses ethanol-46M_RF system to scale on multiple GPU nodes.
```

Comment:
@pszi1ard please share a reference for this input case as well.

Reply:
Here's the upload of you paper's supplementary info: https://zenodo.org/doi/10.5281/zenodo.11087334

The direct link is: https://zenodo.org/records/11087335/files/grappa-46M.tar.bz2